bin/build_package.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * This file is used to compile the assets from an UX package.
+ */
+
+const { parseArgs } = require('node:util');
+const path = require('node:path');
+const fs = require('node:fs');
+const glob = require('glob');
+const rollup = require('rollup');
+const CleanCSS = require('clean-css');
+const { getRollupConfiguration } = require('./rollup');
+
+const args = parseArgs({
+    allowPositionals: true,
+    options: {
+        watch: {
+            type: 'boolean',
+            description: 'Watch the source files for changes and rebuild when necessary.',
+        },
+    },
+});
+
+async function main() {
+    const packageRoot = path.resolve(process.cwd(), args.positionals[0]);
+
+    if (!fs.existsSync(packageRoot)) {
+        console.error(`The package directory "${packageRoot}" does not exist.`);
+        process.exit(1);
+    }
+
+    if (!fs.existsSync(path.join(packageRoot, 'package.json'))) {
+        console.error(`The package directory "${packageRoot}" does not contain a package.json file.`);
+        process.exit(1);
+    }
+
+    const packageData = require(path.join(packageRoot, 'package.json'));
+    const packageName = packageData.name;
+    const srcDir = path.join(packageRoot, 'src');
+    const distDir = path.join(packageRoot, 'dist');
+
+    if (!fs.existsSync(srcDir)) {
+        console.error(`The package directory "${packageRoot}" does not contain a "src" directory.`);
+        process.exit(1);
+    }
+
+    if (fs.existsSync(distDir)) {
+        console.log(`Cleaning up the "${distDir}" directory...`);
+        await fs.promises.rm(distDir, { recursive: true });
+        await fs.promises.mkdir(distDir);
+    }
+
+    const inputScriptFiles = [
+        ...glob.sync(path.join(srcDir, '*controller.ts')),
+        ...(['@symfony/ux-react', '@symfony/ux-vue', '@symfony/ux-svelte'].includes(packageName)
+            ? [path.join(srcDir, 'loader.ts'), path.join(srcDir, 'components.ts')]
+            : []),
+        ...(packageName === '@symfony/stimulus-bundle'
+            ? [path.join(srcDir, 'loader.ts'), path.join(srcDir, 'controllers.ts')]
+            : []),
+    ];
+
+    const inputStyleFile = packageData.config?.css_source;
+    const buildCss = async () => {
+        const inputStyleFileDist = inputStyleFile
+            ? path.resolve(distDir, `${path.basename(inputStyleFile, '.css')}.min.css`)
+            : undefined;
+        if (!inputStyleFile) {
+            return;
+        }
+
+        console.log('Minifying CSS...');
+        const css = await fs.promises.readFile(inputStyleFile, 'utf-8');
+        const minified = new CleanCSS().minify(css).styles;
+        await fs.promises.writeFile(inputStyleFileDist, minified);
+    };
```

Comment:
I don't know if `config.css_source` is used somewhere else, I'm not super fan of having this explicit key when you can simply glob `*.css` files.

Reply:
Only in "a-package"/assets/styles directories right ? :p

Reply:
(i'd like to remove the CSS files to be honest)

Reply:
For a next iteration, I will try to use [Lightning CSS](https://lightningcss.dev/) instead

Reply:
For me: we don't need to build CSS

Reply:
I played with Lightning CSS, it's dope.